### ISSUE TITLE
Pa11y errors on images

### ISSUE THREAD

**Issue by gestchild** [wellcomecollection/wellcomecollection.org#7746]
The [Pa11y dashboard](https://dash.wellcomecollection.org/pa11y/) has started showing errors for missing alt text.

I've had a very quick look and can't find images without alt text, so don't think it is urgent.

Weirdly the src of the images is 'data:image/gif;base64...', which isn't on the page unless loaded without javascript, so looks like it may be to do with lazy loading.

Needs further investigation.
